eslint/babel-eslint-parser/src/convert/index.cts

```diff
@@ -0,0 +1,26 @@
+import convertTokens = require("./convertTokens.cts");
+import convertComments = require("./convertComments.cts");
+import convertAST = require("./convertAST.cts");
+import type { AST, ParseResult } from "../types.cts";
+
+export function convertFile(
```

Comment:
I like to think of TypeScript as "just adding types on top of otherwise untransformed JS code". Given that this file is meant to be CommonJS, would it be possible to do `exports.convertFile = function (...) { ... }`? Or does TS complain?

Reply:
TS will complain about this. When we import, the type of function does not exist.